plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3268,11 +3309,17 @@ public void testOptimize()
                 .collect(toImmutableList());
     }
 
-    private List<String> getAllDataFilesFromTableDirectory(String tableName)
+    protected List<String> getAllDataFilesFromTableDirectory(String tableName)
             throws IOException
     {
         String schema = getSession().getSchema().orElseThrow();
-        Path tableDataDir = getDistributedQueryRunner().getCoordinator().getBaseDataDir().resolve("iceberg_data").resolve(schema).resolve(tableName).resolve("data");
+        Path tableDataDir = getDistributedQueryRunner().getCoordinator()
+                .getBaseDataDir()
+                .resolve("iceberg_data")
+                .resolve(schema)
+                .resolve(tableName)
+                .resolve("data")
+                .toAbsolutePath();
```

Comment:
Looks like `.toAbsolutePath()` is the only change here. Does it matter? why?

Reply:
It's a leftover from refactor i reverted, but forgot about this one. - Reverting